release/models/optical-transport/openconfig-transport-types.yang

```diff
@@ -867,6 +873,20 @@ module openconfig-transport-types {
       50 Gb/s signal";
   }
 
+  identity SFP_DD {
+    base TRANSCEIVER_FORM_FACTOR_TYPE;
+    description
+      "SFP DD pluggable transceiver supporting up to 100 Gb/s signal
+      with 2x50G physical channels.";
+  }
+
+  identity DSFP {
+    base TRANSCEIVER_FORM_FACTOR_TYPE;
+    description
+      "Dual Small form-factor pluggable transceiver supporting up to
+      100 Gb/s signal with 2x50G physical channels.";
```

Comment:
This one has many different electrical options.  How about just pointing to the reference specification?

```suggestion
      "A transceiver implementing the DSFP Transceiver specification.";
      reference "https://dsfpmsa.org/";
```

Reply:
Thanks @dplore, `DSFP` description is also updated following your example.

Reply:
LGTM, thanks